jekyll/_cci2/orb-intro.md

```diff
@@ -35,18 +35,18 @@ Using a private orb enables you to author an orb while ensuring the following:
 
 * your orb cannot be used in a pipeline that does not belong to your organization.
 
-To create a private orb, simply run the `circleci orb create --private <ns/orb>` command in an existing namespace.
+Note that you cannot use private orbs from one organization in another organization's pipelines, regardless of the relationship between organizations. This means that even if you commit code and start a pipeline, and have the necessary membership in both organizations, you can use a private orb from your configuration file, but not from another orb.
 
-By choosing to use a private orb instead of a public orb, you also need to understand certain limitations inherent in using private orbs, which include:
-
-* you will be unable to use the `circleci config validate` command to validate your configuration. You may, however, either paste the content of the orb into the "orbs" stanza of your configuration inline or use the `circleci config validate --orgSlug <your-org-slug> <path/to/config.yml>` command to validate your configuration.
```

Comment:
Just curious - is this part removed because it's on the other page already?

Reply:
@skimke that was a mistake -- fixed